src/H5VL_logi_dataspace.cpp

```diff
@@ -181,24 +192,6 @@ H5VL_log_selections::H5VL_log_selections (hid_t dsid) {
 	ndim = H5Sget_simple_extent_dims (dsid, this->dims, NULL);
 	LOG_VOL_ASSERT (ndim == this->ndim)
 
-	// is the space simple and NULL?
-	if (H5Sis_simple(dsid)) {
```

Comment:
All dataspaces are currently simple.

Reply:
I just now run this line insider of null_space.cpp
```
printf("++++ is null_space simple? %d\n", H5Sis_simple(null_space));
```
and it prints
```
++++ is null_space simple? 0
```